nodejs/nodejs-cloud-run-hello-world/.idea/externalDependencies.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+    <component name="ExternalDependencies">
```

Comment:
probably we don't want to check in this `.idea` folder?

Reply:
This is a requirement of the sample, though the contents of this folder may change.